recipes/acstools/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "acstools" %}
+{% set version = "3.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 254b02018d9d5638235c2bdf930e4752a80435c674f258cd83eeea3fc1f4ed36
+
+build:
+  number: 0
+  skip: True  # [py<36 or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```

Comment:
add within `build` section:
```yaml
entry_points:
  - acs_destripe = acstools.acs_destripe:main
  - acs_destripe_plus = acstools.acs_destripe_plus:main
```

This will allow things to work on Windows.